### PR TITLE
Improved results

### PR DESCRIPTION
**Pull request by ViMuilu**
### Description
Added more tables to result tiles

### Related issues
#138 

### Type of change

- [x] New feature (non-breaking change which adds functionality)

### Changes Made

- More tabels in beaconResultTileDetailsV2
- Disabled Access tags for beacon2 results since beacon2 does not return this data

### Testing

- [x] Needs testing (start an issue or do a follow up PR about it)

### CODE REVIEW

**Comment by ViMuilu** [comment]
i think i made this branch off the beacon2 feature branch thats why there are so many commits. Commits relevant to the pr start from "changed results into tables" 2000a16

File: src/components/BeaconResultTileDetailsV2.vue [hunk 1]
```diff
@@ -16,22 +16,27 @@
...
-              title="Dataset is in public access"
-              ><b>{{ getId(results) }}</b></b-tag
+          <b-table-column label="Id" field="id" v-slot="props">
+            <b-tag class="accessibility-green-tag"
+              ><b>{{ getId(props) }}</b></b-tag
             >
           </b-table-column>
-          <b-table-column label="Access">
-            <template v-slot:default="results">
+          <!-- remove tags if accessibility is not visible in baecon2 --->
```
Comment (blankdots) [inline src/components/BeaconResultTileDetailsV2.vue:35]: lines 35-64 look commented out ? - why is that
if we cannot use in beacon2 maybe we should remove it ?

Comment (blankdots) [reply src/components/BeaconResultTileDetailsV2.vue:35]:
```suggestion
          <!-- remove tags if accessibility is not visible in beacon2 --->
```

File: src/components/BeaconResultTileDetailsV2.vue [hunk 2]
```diff
@@ -148,35 +294,61 @@ export default {
...
     },
     checkForPublicDatasets: function(result) {
-      if (
-        result.info &&
-        result.info.accessType &&
-        result.info.accessType == "PUBLIC"
-      )
-        return true;
+
+      // commented to reduce error messages FIX LATER if possible. Beacon2 does not seem to return access type
```
Comment (blankdots) [inline src/components/BeaconResultTileDetailsV2.vue:298]: if we cannot determine access type for beacon we default to `unknown`, how about that ?
if the keys necessary to determine accesstype exists then we can use the code below

**Review by blankdots — APPROVED** [review]

**Pull request merged**